app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -71,9 +111,44 @@ export default Controller.extend(ModalFunctionality, {
     }
   },
 
+  bindKeyboardShortcuts() {
+    if (!this.mouseTrap) {
+      this.mouseTrap = Mousetrap(
+        document.querySelector("#discourse-modal.bookmark-with-reminder")
```

Comment:
is there any chance this doesn't exist?

Reply:
You mean the modal? Should always exist, this happens in onShow.